python/whylogs/api/store/sqlite_store.py

```diff
@@ -0,0 +1,65 @@
+import logging
+import os
+import sqlite3
+from datetime import datetime
+from typing import Optional, Union
+
+from whylogs.api.store import ProfileStore
+from whylogs.api.store.query import DateQuery, ProfileNameQuery
+from whylogs.core import DatasetProfile, DatasetProfileView
+
+logger = logging.getLogger(__name__)
+
+
+class SQLiteStore(ProfileStore):
```

Comment:
Take a look at my sqlite wrapper: https://github.com/whylabs/whylogs-container/blob/mainline/src/main/kotlin/ai/whylabs/services/whylogs/util/SqliteManager.kt

Had to do a lot of trial and error to improve perf and landed on this